.github/workflows/e2e.yml

```diff
@@ -48,6 +48,13 @@ jobs:
       run: |
         kubectl create -f rte-e2e.yaml
 
+    - name: cluster info
+      run: |
+        kubectl get nodes
+        kubectl get pods -A
+        kubectl describe pod -l name=resource-topology || :
```

Comment:
Why avoid the  ```kubectl describe pod -l name=resource-topology``` call from return with error in case of failure? can the pod has a different name?

Reply:
because we have a race on container start, and I didn't want to complicate the steps adding wais for the container to be Ready - we would need to add and handle timeouts in this case.